e2e/chat.spec.ts

```diff
@@ -178,13 +178,16 @@ describe('chat', () => {
     expect(formattedAmount).toBe(0.2345);
   });
   test('model dropdown interactions', async ({ page }) => {
-    const DEFAULT_MODEL_DISPLAY_NAME = 'DeepSeek RI 67TB';
+    const DEFAULT_MODEL_DISPLAY_NAME = 'DeepSeek R1 67TB';
+    const DEFAULT_MODEL_DESCRIPTION = 'Logical Analysis Engine';
```

Comment:
I added in description to the dropdown on desktop per the mocks so this is now needed